app/build.gradle

```diff
@@ -65,6 +66,10 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    kotlinOptions {
+        jvmTarget = "1.8"
```

Comment:
would be great if we init the property by the following const instead 🤔 
    kotlinOptions {
        jvmTarget = JavaVersion.VERSION_1_8.toString()
    }

Reply:
yep, I will update, thanks man 👍